src/homework14/Box.java

```diff
@@ -0,0 +1,64 @@
+package homework14;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+public class Box <T>{
```

Comment:
`<T>` means any type, but we know that box could hold `Apple` or `Orang`, so `T` must extend `Fruit`.